ui/src/js/components/shared/overview/Overview.js

```diff
@@ -55,6 +53,7 @@ class Overview extends Component {
   */
   componentDidUpdate() {
     this._setExpand();
+    const sectionProps = this.props.labbook || this.props.dataset;
```

Comment:
is there a sectionType props in this component?

```suggestion
     const { props } = this;
     const section = props[props.sectionType];
```